source/user-manual/reference/ossec-conf/syscheck.rst

```diff
@@ -834,6 +825,110 @@ Set the maximum synchronization message throughput.
 | **Allowed values** | Integer number between 0 and 1000000. 0 means disabled. |
 +--------------------+---------------------------------------------------------+
 
+.. _reference_ossec_syscheck_diff:
+
+diff
+^^^^
+
+.. versionadded:: 4.0
+
+The diff settings will be configured inside this tag.
+
+.. code-block:: xml
+
+    <diff>
+      <disk_quota>
+        <enabled>yes</enabled>
+        <limit>1GB</limit>
+      </disk_quota>
+      <file_size>
+        <enabled>yes</enabled>
+        <limit>50MB</limit>
+      </file_size>
+
+      <nodiff>/etc/ssl/private.key</nodiff>
+    </diff>
+
+disk_quota
+""""""""""
+
+.. versionadded:: 4.0
+
+This option can be used to limit the size of the ``queue/diff/local`` folder where Wazuh stores the compressed files used to perform the diff operation when ``report_changes`` is enabled. After reaching this size, alerts will not show the diff information until the size is smaller than the configured limit.
+
+**enabled**
+
+.. versionadded:: 4.0
+
+Specifies whether there will be a disk quota limit or not.
+
++--------------------+---------------------------------------+
+| **Default value**  | yes                                   |
++--------------------+---------------------------------------+
+| **Allowed values** | yes/no                                |
++--------------------+---------------------------------------+
+
+**limit**
+
+.. versionadded:: 4.0
+
+Specifices the limit for the size of the ``queue/diff/local`` folder.
+
++--------------------+---------------------------------------------+
+| **Default value**  | 1GB                                         |
++--------------------+---------------------------------------------+
+| **Allowed values** | Any positive number followed by KB/MB/GB    |
++--------------------------+---------------------------------------+
+
+file_size
+"""""""""
+
+.. versionadded:: 4.0
+
+This option can be used to limit the size of the file which will report diff information with ``report_changes`` enabled. Files bigger than this limit will not report diff information until the size is smaller than the configured limit again.
+
+**enabled**
+
+.. versionadded:: 4.0
+
+Specifies whether there will be a file size limit or not.
```

Comment:
Set the size limit of a file to enable or disable.

Reply:
Done 8cce8d0